src/bp/sdk/botpress.d.ts

```diff
@@ -1579,10 +1554,43 @@ declare module 'botpress/sdk' {
     ): Promise<void>
   }
 
-  export interface PromptConstructable<T> {
+  export interface ExtractionResult {
+    value: any
+    confidence: number
+  }
+
+  export interface ValidationResult {
+    valid: boolean
+    message?: string
+  }
+
+  export interface PromptConstructable {
     new (ctor: any): Prompt
   }
 
+  export interface PromptDefinition {
```

Comment:
Can you move PromptDefinition and PromptConfig where they were previously in the SDK ? This way we can clearly see what was changed